packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -0,0 +1,66 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    borderWidth,
+    controlHeight,
+    fillHoverColor,
+    standardPadding,
+    tableRowBorderColor
+} from '../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('flex')}
+
+    :host {
+        align-items: center;
+        height: calc(${controlHeight} + 2 * ${borderWidth});
+        border-top: calc(2 * ${borderWidth}) solid ${tableRowBorderColor};
+        padding-left: calc(
+            ${standardPadding} * 2 *
+                var(--ni-private-table-group-row-indent-level)
+        );
+    }
+
+    :host(:hover) {
+        background: ${fillHoverColor};
+    }
+
+    :host([expanded='true']) .expander-icon,
+    :host([expanded]) .expander-icon {
+        transform: rotate(90deg);
+    }
+
+    :host([expanded='false']) .expander-icon {
```

Comment:
Are you seeing a case that `expanded` is `false` on the element in the DOM? Since the attribute mode is `boolean`, I would've expected that attribute to be removed when the value is `false`.

Reply:
Yep, I see that `expanded` actually being give the value `false` in the DOM, when it is assigned that boolean value. I haven't figured out a way, yet, to get the visual state to revert back to the "non-expanded" state without the CSS you see here. Apparently `host([expanded])` is a valid selector for `expanded=false`.

Reply:
Never mind. This foolishness was a result of me not binding the attribute properly.